CHANGELOG.md

```diff
@@ -1,7 +1,15 @@
 
 # Changelog
 
-## v0.49.4 (unreleased)
+## v0.49.4
+
+### Breaking changes
+
+- Remove the `CocipGridParams.met_slice_dt` parameter. Now met downselection is handled automatically during contrail evolution. When the `met` and `rad` data passed into `CocipGrid` are not already loaded into memory, this update may make `CocipGrid` slightly more performant.
+- No longer explicitly load `met` and `rad` time slices into memory in `CocipGrid`. This now only occurs downstream when interpolation is performed. This change better aligns `CocipGrid` with other pycontrails models.
+- Remove the `cocipgrid.cocip_time_handling` module. Any useful tooling has been moved directly to the `cocipgrid.cocip_grid` module.
```

Comment:
My warnings 😭 😭 😭 

More seriously---unless I've missed something, the detailed warnings added to address #153 have been lost along with `cocip_time_handling.py`. I think we should keep these warnings, since they address a sharp edge and are referenced in the `CocipGrid` docstring. (Caveats about possible bias because I implemented those changes apply....) `_check_overlap` in `cocip_grid.py` seems like the right place to restore those warnings.

Reply:
I've included this back in the `_set_timesteps` method. 143fdeab259d9a568a8e119778fa287d41c757b8

I've also tweaked one of my new tests to ensure this warning is emitted.